spec/services/meeting_starter_spec.rb

```diff
@@ -20,16 +21,31 @@
       meta_endCallbackUrl: 'http://example.com/meeting_ended',
       'meta_bbb-recording-ready-url': 'http://example.com/recording_ready',
       'meta_bbb-origin-version': 3,
-      'meta_bbb-origin': 'greenlight'
+      'meta_bbb-origin': 'greenlight',
+      setting: 'value'
     }
   end
 
   describe '#call' do
-    it 'calls BigBlueButtonApi with the right params' do
+    let(:room_setting_getter_service) { instance_double(RoomSettingsGetter) }
+
+    before do
+      allow(RoomSettingsGetter).to receive(:new).and_return(room_setting_getter_service)
+      allow(room_setting_getter_service).to receive(:call).and_return({ setting: 'value' })
+    end
+
+    it 'calls BigBlueButtonApi and RoomSettingsGetter with the right params' do
       allow_any_instance_of(BigBlueButtonApi)
         .to receive(:start_meeting)
         .and_return(meeting_starter_response)
 
+      expect(RoomSettingsGetter)
+        .to receive(:new)
+        .with(room_id: room.id, provider: 'greenlight', only_bbb_options: true)
```

Comment:
Because we've stubbed the `RoomSettingsGetter.new` class method we don't have to call its original implementation.
In all of these examples `RoomSettingsGetter.new` will always return `room_setting_getter_service` instance double which will be generated before each test.